charts/sonarqube-dce/README.md

```diff
@@ -1,29 +1,30 @@
 # SonarQube
 
 [SonarQube](https://www.sonarqube.org/) is an open sourced code quality scanning tool.
+The Datacenter Edition (DCE) is aiming for High Availability and Clustering, making it a perfect match for scaling the application with increasing demand.
 
 ## Introduction
 
-This chart bootstraps a SonarQube instance with a PostgreSQL database.
+This chart bootstraps a SonarQube DCE Cluster with a PostgreSQL database.
 
-Please note that this chart only supports SonarQube Community, Developer, and Enterprise Editions.
+Please note that this chart does NOT support SonarQube Community, Developer, and Enterprise Editions.
 
 ## Compatibility
 
 | SonarQube Version | Kubernetes Version | Helm Chart Version |
 |-------------------|--------------------|--------------------|
-| 8.8, 8.9          | 1.19, 1.20, 1.21   | 1.0                |
+| 9.1               | 1.19, 1.20, 1.21   | 0.1                |
 
 ## Installing the chart
 
 To install the chart:
 
 ```bash
 git clone https://github.com/SonarSource/helm-chart-sonarqube.git
-cd helm-chart-sonarqube/charts/sonarqube
+cd helm-chart-sonarqube/charts/sonarqube-dce
 helm dependency update
-kubectl create namespace sonarqube
-helm upgrade --install -f values.yaml -n sonarqube sonarqube ./
+kubectl create namespace sonarqube-dce
+helm upgrade --install -f values.yaml --set ApplicationNodes.jwtSecret=$(echo -n "your_secret" | openssl dgst -sha256 -hmac "your_key" -binary | base64) -n sonarqube-dce sonarqube ./
```

Comment:
Could we autogenerate this jwtSecret if not provided? it seems unnecessary to provide it at runtime